src/OpenLoco/src/S5/S5.cpp

```diff
@@ -319,7 +337,7 @@ namespace OpenLoco::S5
                 fs.writeChunk(SawyerEncoding::runLengthSingle, file.gameState);
             }
 
-            if (file.header.flags & SaveFlags::raw)
+            if (static_cast<uint32_t>(file.header.flags) & static_cast<uint32_t>(SaveFlags::raw))
```

Comment:
this is actually a mistake. It should be `file.header.hasFlags(HeaderFlags::isRaw)`